templates/kubernetes-cluster-with-new-vpc.template

```diff
@@ -75,6 +84,27 @@ Metadata:
 
 # The Parameters allow the user to pass custom settings to the stack before creation
 Parameters:
+  VPCCIDR:
+    Description: VPC CIDR Block
+    Type: String
+    AllowedPattern: "(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})/(\\d{1,2})"
+    ConstraintDescription: must be a valid IP CIDR range of the form x.x.x.x/x.
+    Default: '10.0.0.0/16'
+
+  PrivateSubnetCIDR:
+    Description: CIDR Block for the Private Subnet
+    Type: String
+    AllowedPattern: "(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})/(\\d{1,2})"
+    ConstraintDescription: must be a valid IP CIDR range of the form x.x.x.x/x.
+    Default: '10.0.0.0/19'
+
+  PublicSubnetCIDR:
+    Description: CIDR Block for the Public Subnet
```

Comment:
The description should probably mention that this must be a valid subnet of the VPCCIDR and not conflict with PrivateSubnetCIDR.